### ISSUE TITLE
Uncaught TypeError: Assignment to constant variable.

### ISSUE THREAD

**Issue by huubl** [morehawes/waymark#12]
Hi,

The maps don't load anymore, I get:

```
Uncaught TypeError: Assignment to constant variable.
    at Waymark_Map.get_type (waymark-js.min.js:9:141066)
```

Everything seems to work again when search and replace `for(i in` to `for(var i in` in:

https://github.com/morehawes/waymark/blob/master/assets/dist/waymark-js/js/waymark-js.min.js

**Comment by morehawes** [comment]
Hi @huubl,

Thanks for reaching out. Please ensure you are using the latest version (v0.9.20) and would you be able to provide a link to a page where I can see this happening?

Cheers,

Joe

**Comment by huubl** [comment]
Hi @morehawes,

I use the latest version: 0.9.20. This is a page where this happens:
https://www.verkeerenmeer.nl/examenroutes/amsterdam-noord/

Thanks,
Huub

**Comment by morehawes** [comment]
Hi @huubl,

Thanks for the PR and apologies for not getting to this sooner.

I have merged this and have released version 0.9.21 :)

Cheers,

Joe

**Comment by huubl** [comment]
Thanks @morehawes for merging and releasing a new version this quick!
